ocaml/xapi/xapi_pif.ml

```diff
@@ -451,20 +451,17 @@ let introduce_internal
   pif
 
 (* Assertion passes if network has clusters attached but host has disabled clustering *)
-let assert_no_clustering_enabled ~__context ~network ~host =
+let assert_no_clustering_enabled ~__context ~host =
```

Comment:
I think this should take a pif instead of a network then. You have to be able to reconfigure PIFs that are  not in use by clustering. The name of the function should probably be changed to `assert_no_clustering_enabled_on_pif`